README.md

```diff
@@ -41,5 +41,25 @@ The AWS cloud provider provides the interface between a Kubernetes cluster and A
     <strong><a href="https://kubernetes.github.io/cloud-provider-aws/">See the online documentation here</a></strong>
 </p>
 
-## Note
-* All the EBS volume plugin related logic will be in maintenance mode. For new feature request or bug fixes, please create issue or pull request in [EBS CSI Driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
+## Compatibility with Kubernetes
+
+The AWS cloud provider is released with a specific semantic version that correlates with the Kubernetes upstream version. The major and minor versions are equivalent to the compatible upstream release, and the patch version is reserved to denote subsequent releases of the cloud provider code for that Kubernetes release.
+
+| Kubernetes Version          | Latest AWS Cloud Provider Release Version     |
+|-----------------------------|-----------------------------------------------|
+| v1.19                       | v1.19.0-alpha.1                               |
```

Comment:
Is this table an example or do we intend to keep it up to date? given the description above, should be sufficient to just say
| kubernetes  | cloud provider
| v1.19.x                       | v1.19.y                               |

are compatible

Reply:
also do we want to say that v1.19 is  ONLY compatible with kubernetes 1.19 ? i.e. is that our only guarantee? IMO having a weak guarantee like that is fine, mixing and matching other versions are YMMV

Reply:
I was planning on keeping it up to date, since it might be an easy place to come look at if users are wondering what the latest is/if they are up to date.

Reply:
Yeah updating that version can get tedious if we plan to release often. Maybe we should just put `vMajor.Minor` similar to what @wongma7 suggested.